dataAccessObjects.py

```diff
@@ -153,6 +156,9 @@ def getFileInfo(self,textId):
         else:
             #TODO raise exception
             return None
+    def addLikeAccount(self):
+        sql = 'UPDATE text_data SET text_like_count=text_like_count+1 WHERE text_id="{target_id}"'.format(target_id=str(target_id))
```

Comment:
The variable name should use `lower camel case`
And you hasn't declared the `target_id` above this line.